hw/ip/rstmgr/data/rstmgr.hjson.tpl

```diff
@@ -161,8 +191,17 @@
           resval: "0"
         },
 
+        { bits: "3",
+          hwaccess: "hrw",
+          name: "SW_RESET",
+          desc: '''
+            Indicates when a device has reset due to !!RESET_REQ.
```

Comment:
```suggestion
            Indicates when a device has reset due to software initiated reset request.
```

Reply:
i prefer to reference the register so there's no confusion.
Because unfortunately there are software requested non-device resets.

Reply:
Ohh I see. Got it. Dumb question - what does the preceding `!!` indicate?